Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -60,22 +60,30 @@ fi
 # Checking Ubuntu release to adapt software version to install
 RELEASE_CODENAME=$(lsb_release -c -s)
 
-# translate Mint-codenames to Ubuntu-codenames based on https://www.linuxmint.com/download_all.php
-case ${RELEASE_CODENAME} in
-    wilma | xia)
-        RELEASE_CODENAME='noble'
-        ;;
-    vanessa | vera | victoria | virginia)
-        RELEASE_CODENAME='jammy'
-        ;;
-    una | uma | ulyssa | ulyana | jolnir)
-        RELEASE_CODENAME='focal'
-        ;;
-    tricia | tina | tessa | tara)
-        RELEASE_CODENAME='bionic'
-        ;;
-    elsie)
-        RELEASE_CODENAME='bullseye'
+RELEASE_DISTRIBUTOR=$(lsb_release -i -s)
+case ${RELEASE_DISTRIBUTOR} in
+    LinuxMint)
+        # translate Mint-codenames to Ubuntu-codenames based on https://www.linuxmint.com/download_all.php
+        case ${RELEASE_CODENAME} in
+            wilma | xia)
+                RELEASE_CODENAME='noble'
+                ;;
+            vanessa | vera | victoria | virginia)
+                RELEASE_CODENAME='jammy'
+                ;;
+            una | uma | ulyssa | ulyana | jolnir)
+                RELEASE_CODENAME='focal'
+                ;;
+            tricia | tina | tessa | tara)
+                RELEASE_CODENAME='bionic'
+                ;;
+            elsie)
+                RELEASE_CODENAME='bullseye'
+                ;;
+            *)
+                echo "Unable to map ${RELEASE_CODENAME} to an Ubuntu release"
```

Comment:
```suggestion
                echo "Unable to map '${RELEASE_CODENAME}' to an Ubuntu release. Please report this as a bug at https://github.com/ArduPilot/ardupilot/issues".
```

Suggest giving a confused user a path forward.

Reply:
ITYM, "please create a PR to correct this problem <here>"? :-)

Reply:
I've added words.